### PR TITLE
Extend Domain models

### PR DESCRIPTION
**Pull request by ghost**
Add new models as Domain model fields:
* domain category
* dns provider

and one choice:
* domain type

### CODE REVIEW

File: src/ralph/domains/models/domains.py [hunk 1]
```diff
@@ -44,6 +44,35 @@ class WebsiteType(Choices):
     direct = _('Direct')
 
 
+class DomainType(Choices):
+    _ = Choices.Choice
+
+    none = _('None')
```
Comment (andrzej-jankowski) [inline src/ralph/domains/models/domains.py]: Please delete it :)

**Review by andrzej-jankowski — COMMENTED** [review]
Please add missing API views.

File: src/ralph/domains/models/domains.py (hunk 1 above)
Comment (ghost) [reply src/ralph/domains/models/domains.py]: done

File: src/ralph/admin/sitetrees.py [hunk 2]
```diff
@@ -173,6 +173,8 @@ def section(section_name, app, model):
                 section(_('Contracts'), 'domains', 'DomainContract'),
                 section(_('Registrants'), 'domains', 'DomainRegistrant'),
                 section(_('Owners'), 'accounts', 'RalphUser'),
+                section(_('DNS Provider'), 'domains', 'DNSProvider'),
```
Comment (mkurek) [inline src/ralph/admin/sitetrees.py]: use plural form here

File: src/ralph/domains/models/domains.py [hunk 3]
```diff
@@ -44,6 +44,34 @@ class WebsiteType(Choices):
     direct = _('Direct')
 
 
+class DomainType(Choices):
+    _ = Choices.Choice
+
+    business = _('Business')
+    business_security = _('Business security')
+    technical = _('technical')
```
Comment (andrzej-jankowski) [inline src/ralph/domains/models/domains.py]: To be consistent with the previous ones: Technical

File: src/ralph/admin/sitetrees.py [hunk 4]
```diff
@@ -173,6 +173,8 @@ def section(section_name, app, model):
...
+                section(_('Domain Category'), 'domains', 'DomainCategory'),
```
Comment (mkurek) [inline src/ralph/admin/sitetrees.py]: ditto

File: src/ralph/domains/models/domains.py (hunk 3 above)
Comment (mkurek) [inline src/ralph/domains/models/domains.py]: so lowercase or uppercase? :)

**Review by mkurek — CHANGES_REQUESTED** [review]

**Review by mkurek — APPROVED** [review]

**Review by andrzej-jankowski — APPROVED** [review]

**Pull request merged**